src/App.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import { PressedKey } from './Components/PressedKey';
 
 export const App: React.FC = () => (
   <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
+    <PressedKey />
```

Comment:
There is no need to create a new component. You can store all logic in App.tsx according to the task description